udata/core/activity/models.py

```diff
@@ -97,6 +118,7 @@ def post_save(cls, sender, document, **kwargs):
         if kwargs.get("created"):
             cls.on_create.send(document)
         elif len(changed_fields):
-            cls.on_update.send(document, changed_fields=changed_fields)
+            previous = getattr(document, "_previous_changed_fields", None)
+            cls.on_update.send(document, changed_fields=changed_fields, previous=previous)
```

Comment:
Why getting the `previous` variable here instead of directly in the listener?

Reply:
Hmm it made sense to me that it would be an arg on the `on_update` connecter? We also have this pattern [for owner change](https://github.com/opendatateam/udata/blob/f2b3d41a915f6a9144fbf9d0d1fa9c93a7b52155/udata/core/owned.py#L154).
I can change it if you think it would be better